src/aqua_diagnostics/ocean_drift/plot_hovmoller.py

```diff
@@ -126,7 +126,7 @@ def plot_timeseries(self,
         self.set_data_for_levels()
         self.set_suptitle()
         self.set_title()
-        self.set_description()
+        self.set_description(content="Timeseries of spatially averaged")
```

Comment:
The resulting description is "Timeseries of spatially averaged Pacific Ocean region of climatedt-phase1 IFS-NEMO ssp370" ...I would change to "Timeseries of the spatially averaged Pacific Ocean region for experiment climatedt-phase1 IFS-NEMO ssp370"